Makefile

```diff
@@ -32,7 +34,10 @@ format:
 buildc:
 	@echo "******     Build BPF     ******"
 	@echo
-	gcc ./internal/bpf/bpfWrapper.c -lbpf -c -o ./internal/bpf/bpfWrapper.o
+    ifeq ($(bpf_lib),)
```

Comment:
so this line is supposed to check if libbpf version is <=0.7 right? the set the bpf_flag to -lbpf
On line 15 we check the libxdp version and set libbpf to it. so is this check always being hit or never being hit as we aren't actually comparing versions to anything?